tutorials/monitoring/setup/index.md

```diff
@@ -0,0 +1,209 @@
+
+A Monitoring instance collects data from the monitoring queue and aggregates information from all of the endpoints in the system.
+
+In the ServiceControl Management Utility, click **+ New** and select **Monitoring instance** from the list of options.
+
+![ServiceControl Management Utility - Add new Monitoring instance](scmu-add-new-mon-instance.png)
+
+Scroll down to the **Transport Configuration** section. Select the transport that your NServiceBus system runs on. Depending on the transport selected you may be required to add an additional connection string.
+
+Click the Add button. Your Monitoring instance will start and be listed on the main page of the ServiceControl Management Utility.
+
+![ServiceControl Management Utility - List instance - Monitoring instance](scmu-list-mon-instance.png)
+
+The listing for the Monitoring instance includes a URL link. This URL will be needed when installing ServicePulse below.
+
+NOTE: Creating the Monitoring instance will also create the monitoring queue if it did not already exist. By default this queue is called _Particular.Monitoring_. 
+
+
+
+### Install ServicePulse
+
+ServicePulse is a web application for production monitoring and recoverability. It connects to a Monitoring instance to display monitoring data and to a ServiceControl instance to display recoverability data.
+
+Download and run the latest ServicePulse installer. 
+
+NOTE: If you already have ServicePulse running on your machine you will need to uninstall it and download the latest version in order to be able to configure ServicePulse to connect to a ServiceControl Monitoring instance.
+
+On the ServicePulse Configuration screen ensure that Recoverability is enabled and enter the URL of the ServiceControl instance API. Check the box marked Monitoring and enter the URL of the Monitoring instance API. 
+
+![ServicePulse Installer - ServiceControl configuration](sp-installer-sc-config.png)
+
+You can find the URLs for each instance API in the ServiceControl Management Utility.
+
+![ServiceControl Management Utility - Instance List - Api Addresses Highlighted](scmu-list-api-addresses.png)
+
+Launch ServicePulse and navigate to the Monitoring tab.
+
+![ServicePulse - Monitoring Tab - Empty](sp-monitoring-empty.png)
+
+NOTE: If Monitoring was not enabled during installation, the monitoring tab will not be visible.
+
+
+## Configure NServiceBus endpoints
+
+In order to be monitored by the Particular Service Platform, each endpoint needs to be configured to send data to three different queues: error, audit, and monitoring.
+
+It is recommended to go through this entire process for a single endpoint and ensure that it is working before continuing on to the next endpoint.
+
+
+### Error
+
+Whenever an NServiceBus endpoint is unable to process a message (even after several attempts) it will forward a copy to the error queue. Each environment should contain a single error queue and should not share an error queue with another environment.
```

Comment:
Should we say a copy here? I think that it might be interpreted by some readers that the original messages stays in the input queue of the source endpoint.